pkg/kubelet/cm/container_manager_linux_test.go

```diff
@@ -268,3 +268,35 @@ func TestGetCapacity(t *testing.T) {
 		})
 	}
 }
+func TestNewSystemCgroups(t *testing.T) {
+	cases := []struct {
+		name   string
+		cmName string
+	}{
+		// Not sure what tests should fail here
```

Comment:
After following the code path for possible errors for `newSystemCgroups(...)`, it looks like it points to this [code](https://github.com/opencontainers/runc/blob/182d77c1a9f4462d0a296112833089b4e5c20f1b/libcontainer/cgroups/manager/new.go#L30).

I'd take a look here to get an idea of the error paths for this function.